### ISSUE TITLE
Ensure that we properly handle all sorts of broken inputs

### ISSUE THREAD

**Issue by nikku** [bpmn-io/dmn-migrate#16]
__TLDR:__ When using the utility users must not accidentally swallow errors or warnings.

__What should we do?__

Our [dmn-js](https://github.com/bpmn-io/dmn-js) toolkit does, out of the box, handle all sorts of broken input with appropriate error messages during import:

* binary files (e.g. images)
* non-DMN text files

__Expected Behavior__

* Ensure that the migration utility is en par and handles the issues in the same way.
* Ensure that the migration utility handles warnings appropriately. 

__Why should we do it?__

We advertise the migration utility as a pre-processing step before using dmn-js@8.0.0. 

`diagramXML -> dmn-migrate -> dmn-js#importXML`

In order for that to properly work we must handle errors and warnings appropriately.

**Comment by philippfromme** [comment]
👍
